tests/support/request.js

```diff
@@ -0,0 +1,45 @@
+const { extractBody, extractBinaryBody } = require('./extract.body')
+const { request } = require('http')
```

Comment:
hmmm.  I notice we're exporting our 'call' as 'request' which could be confused with the http request ... should we name it something different, or is that intentional?